modules/aerodyn/src/AeroDyn_Registry.txt

```diff
@@ -339,11 +340,39 @@ typedef	^	MiscVarType	ReKi            WindPos	{:}{:} - -	"XYZ coordinates to que
 typedef	^	MiscVarType	ReKi            WindVel	{:}{:} - -	"XYZ components of wind velocity"	-
 typedef	^	MiscVarType	ReKi            WindAcc	{:}{:} - -	"XYZ components of wind acceleration"	-
 
+# Inflow data storage
+typedef	^	BldInflowType	ReKi	InflowOnBlade	{:}{:}	-	-	"U,V,W at nodes on each blade (note if we change the requirement that NumNodes is the same for each blade, this will need to change)"	m/s
+typedef	^	BldInflowType	ReKi	AccelOnBlade	{:}{:}	-	-	"Wind acceleration at nodes on each blade (note if we change the requirement that NumNodes is the same for each blade, this will need to change)"	m/s
+typedef	^	RotInflowType	BldInflowType    Bld		{:}	-	-	"Blade Inputs"	-
+typedef	^	RotInflowType	ReKi	InflowOnTower	{:}{:}	-	-	"U,V,W at nodes on the tower"	m/s
```

Comment:
How about making the `BldInflowType` not just for blades? You could have this inflow type be for a single element, either blade or tower. Then you'd have RotInflowType contain an array for the blades and a scalar for the tower, something like:
 `Blade(k)%InflowVel` and `Blade(k)%InflowAccel` 
 `Tower%InflowVel` and `Tower%InflowAccel`

Reply:
Good idea!  I'll make that change on PR #2014.

Reply:
Actually, I'll make that on this PR (much simpler).